pynest/nest/server/__init__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+#
+# __init__.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+from .hl_api_server import *
+from .hl_api_client import *
```

Comment:
I would rather not import the client here, but make that into a separate module (see below for details). The rationale behind this is the client is independent of NEST and should be usable also on machines without NEST installed (isn't that the purpose of having a client-server architecture after all?). As submodule imports in Python always import all modules on the way, the current implementation means that an import of the client also always imports `nest`, which i find not desirable.